classes/services/queryBuilders/PKPUserQueryBuilder.inc.php

```diff
@@ -413,7 +431,7 @@ public function getQuery() {
 		$site = \Application::get()->getRequest()->getSite();
 		$primaryLocale = $site->getPrimaryLocale();
 
-		$this->columns[] = 'u.*';
+		$this->columns = ['u.*'];
```

Comment:
If we move to initializing `$this->columns` here, we may as well make it a local variable and not a class member. @NateWr, is this a pattern that's useful elsewhere?

Reply:
I believe it's necessary for the hook, which receives `$this`. That's why the `select()` method is not called until after the hook, so that we can modify the fetched columns.

Reply:
I've modified this part due to something that it's already removed, but I think it makes more sense to keep it this way :)